schema/annotation-source.yaml

```diff
@@ -121,13 +128,17 @@ $defs:
       conclusion:
         $ref_curie: gks.core:Coding
         description: >-
-          The conclusion drawn from the statement proposition, direction, strength, and/or 
-          confidence score.
-      direction:
+          A term that describes a final conclusion about the Statement's subject, based on the statement 
+          proposition, confidence level, and/or evidence direction and strength assessments.  
+      evidence_direction:
         type: string
-        enum: [ "supports", "uncertain", "refutes" ]
+        enum: [ "supports", "uncertain", "disputes" ]
```

Comment:
'disputes' vs 'refutes' . . .  FWIW I think 'disputes' better captures the idea that the evidence goes against the target proposition ('refutes' to me sounds like that evidence is sufficient to definitively refute it).

Reply:
Ugh. I think the domain experts are going to struggle with this (not the specific terms). I think they consider that they have evidence that supports a variation as NOT being pathogenic and thus being Benign.  But I digress (again). We have chosen this course and will stick with the idea that asserting a proposition is "FALSE" is the basis for showing some level of confidence in the "opposing proposition" .  

Maybe we could use the term `opposes` to suggest that the assertion is opposing the proposition.  

Also, we should no use the term `evidence` in the attribute name because it is not about the `evidence`. It is about the assertion in relation to the proposition (regardless of the evidence). Even though the evidence should most certainly support with the `assertion_direction` whatever direction it may be.

Reply:
I agree with the concern @mbrush raises here, and I really like `opposes` as the solution. 👍

Reply:
I am good with 'disputes' or 'opposes'.